app/index.js

```diff
@@ -1,262 +1,640 @@
+            viewsArray[0].viewObj.style.display = 'inline';
+            viewsArray[1].viewObj.style.display = 'inline';
+            console.log("---> Displayed thank you");
+            // Display clockface
+            setTimeout( () => {
+                viewDisplay(viewsArray[0]);
+            }, 2000);
+            break;
+        case 'svg_stop_survey':
+            // Display stop survey for 2s
+            viewsArray[0].viewObj.style.display = 'inline';
+            viewsArray[3].viewObj.style.display = 'inline';
+            console.log("---> Displayed survey stopped");
+            // Display clockface
+            setTimeout( () => {
+                viewDisplay(viewsArray[0]);
+            }, 2000);
+            break;
+        case 'question':
+            const questionLength = view.viewQuestion.iconText.length;
+            let allButtonLocations = ["left", "right", "center", "bottom"];
+
+            view.viewObj.style.display = "inline";
+            document.getElementById("question-text" + questionLength.toString()).text = view.viewQuestion.questionText;
+            document.getElementById("question-second-text" + questionLength.toString()).text = view.viewQuestion.questionSecondText;
+            console.log("---> Displayed text question");
+            
+            const objectsLocation = returnButtonLocations(questionLength);
+            const buttonsLocations = Object.keys(objectsLocation);
+            // Hide all buttons
+            hideAllButtons();
+            console.log("---> Hidden all buttons");
+            // Display needed buttons
+            view.viewQuestion.iconText.forEach((text, index) => {
+                // Show button
+                document.getElementById("new-button-" + buttonsLocations[index] + questionLength.toString()).style.display = "inline";
+                // Show circle
+                document.getElementById("circle-" + buttonsLocations[index] + questionLength.toString()).style.fill = view.viewQuestion.iconColors[index];
+                // Show image
+                document.getElementById("image-" + buttonsLocations[index] + questionLength.toString()).href = view.viewQuestion.iconImages[index];
+                // Show text
+                document.getElementById("button-text-" + buttonsLocations[index] + questionLength.toString()).text = view.viewQuestion.iconText[index];
+            });
+            console.log("---> Displayed needed buttons");
+            break;
+        default:
+            break;
+    }
+    vibration.start("bump");
+}
+
+// ------- BUZZ SELECTOR -------------------
+
+const errorLabel = document.getElementById("errorLabel");
+const bodyErrorLabel = errorLabel.getElementById("copy");
+
+// define what at what hour of the day each buzz option would buzz at
 const buzzOptions = {
     0: [],
-    1: [9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21],
+    1: [9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
     2: [9, 11, 13, 15, 17, 19, 21],
```

Comment:
I'm assuming you deleted 20:00, and 21:00 to not disturb people in Option 1. Are you also going to do the same for option 2, and 3